src/ci/bin/test.inc.sh

```diff
@@ -660,9 +660,11 @@ cromwell::private::create_centaur_variables() {
     case "${CROMWELL_BUILD_CENTAUR_TYPE}" in
         "${CROMWELL_BUILD_CENTAUR_TYPE_INTEGRATION}")
             CROMWELL_BUILD_CENTAUR_READ_LINES_LIMIT=512000
+            CROMWELL_BUILD_CENTAUR_MAX_WORKFLOW_LENGTH="10 hours"
```

Comment:
Is this just explicitly stating the default from [here](https://github.com/broadinstitute/cromwell/blob/e7b8f13443ee88f281b305fcaf30dc4deaafec11/centaur/src/main/resources/reference.conf#L43-L43) so that you can provide a different value below ("90 minutes")?